tests/py/unit/fixtures.py

```diff
@@ -492,6 +492,99 @@
     "isUpdated": True,
 }
 
+GRAPHENE_ZIGZAG_NANORIBBON_PASSIVATED = {
```

Comment:
Nit: we can put this to Standata and import here - will be easier to visualize, etc.